src/overlays/actors/ovl_Obj_Switch/z_obj_switch.c

```diff
@@ -322,6 +322,8 @@ void ObjSwitch_Init(Actor* thisx, GlobalContext* globalCtx) {
     if (this->dyna.actor.params >> 7 & 1) {
         ObjSwitch_EyeFrozenInit(this);
     } else if (type == OBJSWITCH_TYPE_FLOOR || type == OBJSWITCH_TYPE_FLOOR_RUSTY) {
+        //! @bug if the subtype is OBJSWITCH_SUBTYPE_RESET_INVERTED, the switch should be Down or Up according to
+        //! `!switchFlagSet` instead of `switchFlagSet`
```

Comment:
I dont understand this, wouldnt you want the switch to be down (pressed) if the switch flag is set? So isnt this right?

Reply:
The `OBJSWITCH_SUBTYPE_RESET_INVERTED` subtype specifically actually has the switch flag set when it's released and cleared when it's pressed, is the documentation on that in the actor header unclear?

This bug is specific to `OBJSWITCH_SUBTYPE_RESET_INVERTED` which isn't used in game iirc

Reply:
i completely glazed over it being specific to `OBJSWITCH_SUBTYPE_RESET_INVERTED`. But I think we can still improve on the wording of the bug description to make it more clear

maybe something like "this condition does not account for `OBJSWITCH_SUBTYPE_RESET_INVERTED` which expects the switch flag to be set for the up position (and vice versa)"

Reply:
That wording is better imo, I'll wait to have @Dragorn421's input on that idea though

Maybe something like that to be more precise? "This condition does not account for the subtype, that makes ``OBJSWITCH_SUBTYPE_HOLD_INVERTED`` have the wrong init since the switch flag is supposed to be set when the switch is released"

Reply:
I think overall a high level explanation of the problem that gets the point across quickly is good enough. editing my previous idea even further:
"This condition does not account for `OBJSWITCH_SUBTYPE_HOLD_INVERTED` which expects the relationship between the switch position and the switch flag to be inverted"
that way we arent confuysing the reader about down vs up and set vs not set